benchmarks/asv.conf.json

```diff
@@ -29,7 +29,7 @@
     "matrix": {
         "numpy": ["1.8.2"],
         "Tempita": ["0.5.2"],
-        "Cython": ["0.23.4"],
+        "Cython": ["0.25.2"],
```

Comment:
@droyed The source of the merge conflict is the older Cython version used here. If you can update that string to `0.27.3` (we recently bumped the min version used), this should resole the issue (@pv could be quite busy & that should be a straightforward fix).